# Generated by Django 2.2.5 on 2019-11-05 09:02

from django.conf import settings
from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('faulty_logging', '0003_remove_faultylogging_next_follow_up'),
    ]

    operations = [
        migrations.AlterField(
            model_name='faultylogging',
            name='assigned_to',
            field=models.ManyToManyField(blank=True, to=settings.AUTH_USER_MODEL),
        ),
        migrations.AlterField(
            model_name='faultylogging',
            name='faulty_close_date',
            field=models.DateField(default=django.utils.timezone.now),
            preserve_default=False,
        ),
    ]
